Task6/css/button.css

```diff
@@ -0,0 +1,63 @@
+.ts6-field {
+    display: flex;
+    align-items: flex-end;
+}
+
+.ts6-button__field {
+    border: 0;
+    padding: 0;
+    border-radius: 5px;
+    color: #ffffff;
+    font-weight: bold;
+    margin: 0 5px 8px 0;
+    cursor: pointer;
+    outline: none;
+}
+
+.ts6-button__field:hover {
+    background: #a6a3a5;
+    color: #33292e
+}
```

Comment:
You can achieve more beautiful result via using CSS function like `brightness()`.

Reply:
Done.